src/renderer/visualizer/circle-visualizer.ts

```diff
@@ -1,3 +1,4 @@
+import { setTimeout } from "timers";
```

Comment:
Is this needed? JS has `setTimeout` build into it :D

Reply:
I thought this too! But for some reason VSCode added it for me, so I thought it knew better